lib/collection/request.js

```diff
@@ -33,19 +33,25 @@ _.inherit((
         // this constructor is intended to inherit and as such the super constructor is required to be executed
         Request.super_.apply(this, arguments);
 
-        // in case definition object is missing, there is no point moving forward
-        if (!options) { return; }
+        // in case definition object is missing, create an empty one
+        options = options || {};
```

Comment:
I also see in other properties that options is not initialised to an object if missing. instead, it is taken care in update function using `_.get`

https://github.com/postmanlabs/postman-collection/blob/66ad51b62dcb2f519b379d027b6864dbab4684f0/lib/collection/header.js